gcloud_bigtable/google/bigtable/admin/cluster/v1/bigtable_cluster_service_api.py

```diff
@@ -0,0 +1,262 @@
+      'https://www.googleapis.com/auth/cloud-bigtable.admin',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin.cluster',
+      'https://www.googleapis.com/auth/cloud-platform'
+  ]
+
+  def __init__(
+      self, service_path=_SERVICE_ADDRESS, port=_DEFAULT_SERVICE_PORT,
+      channel=None, ssl_creds=None, scopes=_ALL_SCOPES,
+      is_idempotent_retrying=True, max_attempts=3, timeout=30):
+
+    self.defaults = api_callable.ApiCallableDefaults(
+      timeout=timeout, max_attempts=max_attempts,
+      is_idempotent_retrying=is_idempotent_retrying)
+
+    self.stub = api_utils.create_stub(
+      bigtable_cluster_service_pb2.beta_create_BigtableClusterService_stub, service_path, port,
+      ssl_creds=ssl_creds, channel=channel, scopes=scopes)
+
+  def __enter__(self):
+    return self
+
+  def __exit__(self, type, value, traceback):
+    self.close()
+
+  def close(self):
+    del self.stub
+
+  # Properties
+  @property
+  def channel(self):
+    return self.channel
+
+  # Page descriptors
+
+  # Service calls
+  def list_zones(self, name="", **kwargs):
+    """Lists the supported zones for the given project."""
+    list_zones_request = bigtable_cluster_service_messages_pb2.ListZonesRequest(
+        name=name, **kwargs)
+    return self.list_zones_callable()(list_zones_request)
+
+  def list_zones_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.ListZones,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def get_cluster(self, name="", **kwargs):
+    """Gets information about a particular cluster."""
+    get_cluster_request = bigtable_cluster_service_messages_pb2.GetClusterRequest(
+        name=name, **kwargs)
+    return self.get_cluster_callable()(get_cluster_request)
+
+  def get_cluster_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.GetCluster,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def list_clusters(self, name="", **kwargs):
+    """
+    Lists all clusters in the given project, along with any zones for which
+    cluster information could not be retrieved.
+    """
+    list_clusters_request = bigtable_cluster_service_messages_pb2.ListClustersRequest(
+        name=name, **kwargs)
+    return self.list_clusters_callable()(list_clusters_request)
+
+  def list_clusters_callable(
+      self, is_retrying=None, max_attempts=None):
+    return api_callable.idempotent_callable(
+        self.stub.ListClusters,
+        is_retrying=is_retrying,
+        max_attempts=max_attempts,
+        defaults=self.defaults)
+
+  def create_cluster(self, name="", cluster_id="", cluster=bigtable_cluster_data_pb2.Cluster(), **kwargs):
+    """
+    Creates a cluster and begins preparing it to begin serving. The returned
```

Comment:
When I wrote docstrings by hand I mostly just copied the docs from the `.proto` files.

However, for some things like `create_cluster`, there is more information about the acceptable arguments, e.g. the [cluster name][1] is of the form `[a-z][-a-z0-9]*` (I also know must be between 4-30 chars, but that isn't in the `.proto` but maybe should be?)

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/23d6e474b9c2d27d85e450197e79d5333ac790e5/bigtable-protos/src/main/proto/google/bigtable/admin/cluster/v1/bigtable_cluster_data.proto#L62